zipkin-lens/.env.production

```diff
@@ -0,0 +1,2 @@
+VITE_UI_URL=/zipkin
```

Comment:
I don't think it is a good idea to check in env specific files.

Reply:
This actually sets the ui url for the frontend. I'm not sure what you mean with "check in env specific files".